.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+<<<<<<< Updated upstream
```

Comment:
I don't think you meant to commit and push it 😅

In the future, make sure to be precise with files you stage and commit 🙂 Avoid `git add .` unless you are 100% sure about what you are committing.

Personally, I use VSCode to stage files and create commits. This way I know what I am committing 😄